spec/models/user_spec.rb

```diff
@@ -22,7 +22,9 @@
       let(:auth) { mock_auth user }
 
       it 'creates a new User record' do
-        expect{ User.from_omniauth(auth) }.to change{ User.count }.by(1)
+        expect do
```

Comment:
This is a false test — `User.from_omniauth` doesn't create a record anymore. Please just remove it, not just hack around to make it green.

Tests are also documentation of functions, it's an absolute antipattern to make incorrect statements here.